great_expectations/datasource/fluent/spark_datasource.py

```diff
@@ -92,6 +92,10 @@ def _validate_dataframe(cls, dataframe: pyspark.DataFrame) -> pyspark.DataFrame:
     def test_connection(self) -> None:
         ...
 
+    @property
+    def batch_request_options(self) -> tuple[str, ...]:
```

Comment:
Could you comment on why this was added?

Reply:
@anthonyburdi It is an interface method.  `Python` does not enforce it, but the `Pandas` equivalent does provide this implementation -- we should try keeping it clean.  Thanks.

Reply:
Thank you for clarifying, that makes sense!